assets/screens/input_maps.onj

```diff
@@ -7,7 +7,7 @@ defaultInputMap: [
         modifiers: [],
         action: $ToggleFullscreenKeyAction {
             width: 700,
-            height: 500
+            height: 700*9/16
```

Comment:
not a big deal, but parens what be useful here to make the order of operations clearer